src/objectTranslations/GetStartedChecklistItem__mdt-iw.objectTranslation

```diff
@@ -3,22 +3,145 @@
     <caseValues>
         <article>None</article>
         <plural>false</plural>
-        <value><!-- NPSP Get Started Admin Checklist Item --></value>
+        <value><!-- NPSP Get Started Checklist Item --></value>
     </caseValues>
     <caseValues>
         <article>Definite</article>
         <plural>false</plural>
-        <value><!-- NPSP Get Started Admin Checklist Item --></value>
+        <value><!-- הNPSP Get Started Checklist Item --></value>
```

Comment:
This ה character turns up in a few spots that I wouldn't expect it to in the Hebrew translations alongside english text. All have been commented out so far.

Reply:
@bdvorachek  I've seen that in places as well. Technically we don't support Hebrew as a supported languages, but the files are still in our repo because it was started at one point.